aten/src/ATen/native/vulkan/Vulkan.cpp

```diff
@@ -1290,7 +1290,7 @@ class VulkanTensor::Impl final {
   VkDeviceSize buffer_size_for_sizes(std::vector<int64_t> sizes) const {
     const auto d = sizes.size();
     const auto numel = std::accumulate(
```

Comment:
const int64_t numel and then decltype type later or C++ style cast instead of C-style cast